tests/tests.py

```diff
@@ -293,21 +321,40 @@ def test_rendering_unicode(self):
         output = self.render_template(template_string, context)
         self.assertEqual(output, "äüßéô")
 
-    def test_escaping_special_characters(self):
+    def check_escape(self, value: str, expected: str):
         template_string = "{{ value | latex_escape }}"
-        context = {"value": "&$%#_{}"}
+        context = {"value": value}
         output = self.render_template(template_string, context)
-        expected = "\\&\\$\\%\\#\\_\\{\\}"
         self.assertEqual(output, expected)
 
+    def test_escape_no_space(self):
```

Comment:
Is there some code repetition here? Why don't we use the unicode characters from the `constants.py` module? Or ditch the `constants.py` module and copy all constants into the `tests.py` module for better readability.